app/views/application/_radius_details.html.erb

```diff
@@ -8,9 +8,9 @@
     <% if current_organisation.locations.empty? %>
       <p>Your RADIUS secret keys will be generated when you <%= link_to 'add your first IP address', new_ip_path, class: "govuk-link" %></p>
     <% else %>
-    <p> Your RADIUS secret keys are shown under <%= link_to "your organisation's locations", ips_path, class: "govuk-link" %></p>
+      <p> Your RADIUS secret keys are shown under <%= link_to "your organisation's locations", ips_path, class: "govuk-link" %></p>
     <% end %>
-    <p> We recommend connecting to one RADIUS server from each region: </p>
```

Comment:
Should we be explicit about the fact that you have to use ALL of the ips for failover?

Reply:
Er, I think we are, in this PR:
> We recommend using all of our RADIUS servers, so that you'll be protected from a single server being unavailable.

I wonder if the OLD/NEW screenshots weren't particularly clear!

Reply:
oh right I see it's on the other page.
We don't want to change this message that you removed to encourage people as they look at the list?

Reply:
It's within this box - Pippa recommended having it as the first thing people see within that section:

![image](https://user-images.githubusercontent.com/429326/52797143-2c0b0900-306d-11e9-87d4-61a720575ca9.png)